reflex/.templates/jinja/custom_components/README.md.jinja2

```diff
@@ -0,0 +1,3 @@
+# {{ module_name }}
```

Comment:
Let's include the example app in the README as well like gradio: https://huggingface.co/spaces/freddyaboulton/gradio_calendar

We can include the installation code as well as the demo app code. Perhaps if we want to be smart, we can auto-generate it on `reflex component build`, or we can just let them do it. But should have an installation / usage examples.

We should also list the props in the readme, but they may have to do that manually. There's way to auto-generate it similar to how we do on our main site, so we could do that also..

Reply:
Maybe the only thing we need to add is the `pip install` command, agreed everything else would be a pain to auto-generate